### DISCUSSION TITLE
Can't find /etc/nsm/securityonion.conf / Basic Folder do not exist

### DISCUSSION THREAD

**Discussion by Xen0r** [Security-Onion-Solutions/securityonion#3819] [Unsupported Versions] [answered]
Hi,

I just installed security onion in Standalone Mode on an HPE Proliant DL20. I learned that you can install Security Onion onto Ubuntu, if I got that right. Whatsoever I installed it directly, meaning there should be a CentOs underlying.
My Problem is this: I can't reach the web interface. It should have the management IP address. But everytime I try
to reach it, I get the Error that the address can't be found and it changes automatically into _https://seconion_

Seconion is the hostname of the machine.

So as I can't reach the Web Frontend, i tried to do some configuration over the cli. There was also no desktop environment installed during or after the installation.

With `so-status` I can see that every service is running, but when I look for example for Suricata in the _/etc/nsm/suricata_ Folder
I have the problem that the _/nsm_ Folder doesn't even exists???

The Server should be running as a NIDS in my Company. Therefore I need the software suricata. 

I hope someone can help me with this. Since I'm only 3 years in the IT-Field and never worked with Security Onion before.

**Comment by TOoSmOotH** [comment, marked as answer]
I would start here: https://www.youtube.com/playlist?list=PLljFlTO9rB155aYBjHw2InKkSMLuhWpxH

**Reply by Xen0r** [reply]
Hey, thank you for your answer. I figured everything out. And its running just fine. The problem that I had was looking at the documentation for the version 16.4 instead of 2.3, which was the version that I had.

**Comment by usmanaa10** [comment]
any solution?

**Reply by dougburks** [reply]
@usmanaa10 This discussion has been marked as answered. See the answer above.

**Reply by usmanaa10** [reply]
Thats not answer to my question, I am looking for file but no securityonion.config file has been found

**Reply by dougburks** [reply]
As mentioned in the answer above, there is no securityonion.conf or securityonion.config in the current version of Security Onion. If you are looking at documentation that references securityonion.conf, then that documentation is for an older version of Security Onion. Make sure you are looking at the current version of the documentation:
https://docs.securityonion.net/en/2.3/index.html

This discussion has already been marked as answered. If you have further questions or problems, please start a separate discussion rather than replying to this one.
